model/registry.go

```diff
@@ -36,10 +36,12 @@ type Config struct {
 }
 
 // Registry of the configuration objects
+// Modification of object pointers violates thread-safety
```

Comment:
The comment is not entirely clear to me.
Do you mean that client should not modify the object pointed by the *Config returned by Get() and List() or should not modify the object pointed by the *Config passed to Put() after calling Put() or all of them or something else ?